lib/src/model/puzzle/puzzle_preferences.dart

```diff
@@ -37,8 +44,19 @@ class PuzzlePrefState with _$PuzzlePrefState {
   const factory PuzzlePrefState({
     required UserId? id,
     required PuzzleDifficulty difficulty,
+
+    /// If `true`, will show next puzzle after successful completion. This has
+    /// no effect on puzzle streaks, which always show next puzzle. Defaults to
+    /// `false`.
+    required bool autoNext,
```

Comment:
Since the field is `required` the `fromJson` method would fail with previously saved data from the current version of the app. 
Making it non required would prevent any issue.

Reply:
Removed `required`, added `@Default(false)` as required by freezed.